core/src/main/kotlin/org/evomaster/core/search/gene/sql/SqlNullable.kt

```diff
@@ -49,6 +63,76 @@ class SqlNullable(name: String,
         }
     }
 
+    override fun archiveMutation(randomness: Randomness, allGenes: List<Gene>, apc: AdaptiveParameterControl, selection: GeneMutationSelectionMethod, impact: GeneImpact?, geneReference: String, archiveMutator: ArchiveMutator, evi: EvaluatedIndividual<*>, targets: Set<Int>) {
+        if(!archiveMutator.enableArchiveMutation()){
+            standardMutation(randomness, apc, allGenes)
+            return
+        }
+
+        val preferPresent = if (!archiveMutator.applyArchiveSelection() || impact == null || impact !is SqlNullableImpact) true
+                    else {
+            !impact.presentImpact.run {
+                this.timesToManipulate > 5
```

Comment:
this seems repeated at least twice. should put 5 in a meaningfully named variable, or at least provide some comments